app/views/shared/_modal.html.erb

```diff
@@ -1,6 +1,6 @@
 <%# locals: (content:) -%>
 <%= turbo_frame_tag "modal" do %>
-  <dialog class="bg-white border border-alpha-black-25 rounded-2xl max-h-[648px] max-w-[580px] w-full shadow-xs h-fit" data-controller="modal" data-action="click->modal#clickOutside">
+  <dialog class="bg-white border border-alpha-black-25 rounded-2xl max-h-[648px] max-w-[580px] w-min-content shadow-xs h-fit" data-controller="modal" data-action="click->modal#clickOutside">
```

Comment:
Modal width is now dictated by its contents, which makes it more flexible.

The modal for this feature is quite small — in contrast to the existing modals which are quite wide.